cordova-lib/src/PluginInfo.js

```diff
@@ -288,6 +293,19 @@ function PluginInfo(dirname) {
             return ret;
         });
     };
+
+    self.getFilesAndFrameworks = getFilesAndFrameworks;
+    function getFilesAndFrameworks(platform) {
+        var items = [];
+        // Please avoid changing the order of the calls below, files will be
+        // installed in this order.
+        items = items.concat(self.getSourceFiles(platform));
+        items = items.concat(self.getHeaderFiles(platform));
+        items = items.concat(self.getResourceFiles(platform));
+        items = items.concat(self.getFrameworks(platform));
+        items = items.concat(self.getLibFiles(platform));
```

Comment:
Tip: `concat` actually accepts any number of arguments and will concat all in one go.

Can also do: `[].concat.apply([], [ self.getSourceFiles(..), self.getHeaderFiles(..), ...])`

Which I think looks nicest.